tests/chainer_tests/functions_tests/loss_tests/test_softmax_cross_entropy.py

```diff
@@ -370,19 +372,26 @@ def check_forward(self, x_data, t_data, class_weight):
             testing.assert_allclose(
                 loss_expect, li, **self.check_forward_options)
 
-    @condition.retry(3)
     def test_forward_cpu(self):
-        with chainer.using_config('use_cudnn', self.use_cudnn):
```

Comment:
Why did you remove this argument?

Reply:
Because, this is cpu test.

Reply:
Is it better to check if `use_cudnn` does not affect its behavior event when cup is used?